jcommune-view/jcommune-web-controller/src/main/java/org/jtalks/jcommune/web/dto/GroupDto.java

```diff
@@ -27,89 +27,64 @@
  *
  */
 public class GroupDto {
+    private static int DISPLAY_USER_LIMIT= 20;
+
     private long id;
     private String name;
+    private List<UserDto> users;
 
-    /**
-     * Instantiates new GroupDto object based on the Group object
-     * @param group source data for the DTO object
-     */
     public GroupDto(Group group) {
         this.id = group.getId();
         this.name = group.getName();
+        this.users = UserDto.convertToUserDtoList(group.getUsers(), UserDto.BY_NAME_COMPARATOR, DISPLAY_USER_LIMIT);
```

Comment:
It's not good idea. Group 'Registered users' contain over 40k users which will be retrieved from DB. You could better write a hql query to retrieve a limited list of users from given group.